MAINTAINERS.md

```diff
@@ -109,12 +113,16 @@ If you are an owner of the organization, you can see an automated list [here](ht
   [19 October 2023](https://github.com/tldr-pages/tldr/issues/11075) — [24 October 2023](https://github.com/tldr-pages/tldr/issues/11202)
 - Isaac Vicente ([@isaacvicente](https://github.com/isaacvicente)):
   [20 September 2023](https://github.com/tldr-pages/tldr/issues/10737) — [29 December 2023](https://github.com/tldr-pages/tldr/issues/11918)
-- Vitor Henrique ([@vitorhcl](https://github.com/vitorhcl)): [18 December 2023](https://github.com/tldr-pages/tldr/issues/11771) — [21 January 2024](https://github.com/tldr-pages/tldr/issues/12094)
+- Vitor Henrique ([@vitorhcl](https://github.com/vitorhcl)):
+  [18 December 2023](https://github.com/tldr-pages/tldr/issues/11771) — [21 January 2024](https://github.com/tldr-pages/tldr/issues/12094)
 - Geipro/Proscream ([@Geipro)](https://github.com/Geipro)):
   [19 November 2019](https://github.com/tldr-pages/tldr/issues/3592) — [27 March 2024](https://github.com/tldr-pages/tldr/issues/12209) (Removed during 2FA enforcement)
-- Ruben Vereecken ([@rubenvereecken](https://github.com/rubenvereecken)): [18 January 2018](https://github.com/tldr-pages/tldr/issues/1878#issuecomment-358610454) — [27 March 2024](https://github.com/tldr-pages/tldr/issues/12209) (Removed during 2FA enforcement)
-- Fazle Arefin ([@fazlearefin](https://github.com/fazlearefin)): [09 February 2024](https://github.com/tldr-pages/tldr/issues/12227) — [2 April 2024](https://github.com/tldr-pages/tldr/issues/12595#issuecomment-2031015230)
-- Alejandro Cervera ([@tricantivu](https://github.com/tricantivu)): [4 January 2024](https://github.com/tldr-pages/tldr/issues/11989) — [3 April 2024](https://github.com/tldr-pages/tldr/issues/12594)
+- Ruben Vereecken ([@rubenvereecken](https://github.com/rubenvereecken)):
+  [18 January 2018](https://github.com/tldr-pages/tldr/issues/1878#issuecomment-358610454) — [27 March 2024](https://github.com/tldr-pages/tldr/issues/12209) (Removed during 2FA enforcement)
+- Fazle Arefin ([@fazlearefin](https://github.com/fazlearefin)):
+  [09 February 2024](https://github.com/tldr-pages/tldr/issues/12227) — [2 April 2024](https://github.com/tldr-pages/tldr/issues/12595#issuecomment-2031015230)
```

Comment:
```suggestion
  [09 February 2024](https://github.com/tldr-pages/tldr/issues/12227) — [2 April 2024](https://github.com/tldr-pages/tldr/issues/12595)
```

Lets use this opportunity to remove linking to a specific comment

Reply:
We do allow linking to comments in situations where people are invited/re-invited informally i.e. in PRs or discussion in chatroom, etc. But since this is a formal case, yeah we can remove this.